mobile-client/app/local.properties

```diff
@@ -0,0 +1,8 @@
+## This file must *NOT* be checked into Version Control Systems,
+# as it contains information specific to your local configuration.
+#
+# Location of the SDK. This is only used by Gradle.
+# For customization when using a Version Control System, please read the
+# header note.
+#Mon Jun 15 02:41:14 WAT 2020
+sdk.dir=/home/durrell/Android/Sdk
```

Comment:
Use a relative path here and not an absolute path. I don't think this should be part of you commit.